architecture/http_remote_command_execution.adoc

```diff
@@ -9,4 +9,53 @@
 
 toc::[]
 
-== Overview
\ No newline at end of file
+== Overview
+
+OpenShift takes advantage of a feature built into Kubernetes to support executing commands in containers. This is implemented using HTTP along with a multiplexed streaming protocol such as link:http://www.chromium.org/spdy[SPDY] or link:https://http2.github.io/[HTTP/2].
```

Comment:
Also, there are other docs using Markdown syntax `[text](link)` that should probably be converted